backend/sqlc/study-queries.go

```diff
@@ -0,0 +1,165 @@
+package sqlc
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/bcc-code/brunstadtv/backend/batchloaders"
+	"github.com/bcc-code/brunstadtv/backend/common"
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+	"gopkg.in/guregu/null.v4"
+)
+
+// GetTopics returns studies
+func (q *Queries) GetTopics(ctx context.Context, ids []uuid.UUID) ([]common.StudyTopic, error) {
+	topics, err := q.getTopics(ctx, ids)
+	if err != nil {
+		return nil, err
+	}
+	return lo.Map(topics, func(t getTopicsRow, _ int) common.StudyTopic {
```

Comment:
Would it make sense in this cases to use the parallel version of lo?

Reply:
I don't think it would matter much. It's not waiting on any IO, just iterating through a list of items. 

Might be less efficient to split it into multiple threads.